alerter.c

```diff
@@ -11,21 +13,43 @@ int networkAlertStub(float celcius) {
     return 200;
 }
 
-void alertInCelcius(float farenheit) {
+int networkAlertReal(float celcius) {

```

Comment:
This seems to be unreachable code or not used!! if not used then it should be cleaned up....